testing/perl-config-properties/APKBUILD

```diff
@@ -0,0 +1,47 @@
+# Contributor: Andreas Schipplock <andreas@schipplock.de>
+# Maintainer: Andreas Schipplock <andreas@schipplock.de>
+pkgname=perl-config-properties
+_pkgreal=Config-Properties
+pkgver=1.80
+pkgrel=0
+pkgdesc="Read and write property files"
+url="http://search.cpan.org/dist/Config-Properties/"
+arch="noarch"
+license="GPL PerlArtistic"
+cpandepends=""
```

Comment:
If lines are empty (or empty as a result of containing empty variables, you can safely omit then.